src/Tribe/Attendees_Table.php

```diff
@@ -70,7 +72,9 @@ public function __construct( $args = [] ) {
 
 		// Fetch the event Object
 		if ( ! empty( $_GET['event_id'] ) ) {
-			$this->event = get_post( absint( $_GET['event_id'] ) );
+			$event_id    = filter_var( $_GET['event_id'], FILTER_VALIDATE_INT );
+			$event_id    = Event::filter_event_id( $event_id );
+			$this->event = get_post( absint( $event_id ) );
```

Comment:
If the `Event::filter_event_id` method returns `?int`, then there should be no need for the `absint` call.